### ISSUE TITLE
"There was an error processing your Goggle"

### ISSUE THREAD

**Issue by digital-carver** [brave/goggles-quickstart#33]
I created a new Goggle for the Julia programming language, at https://github.com/digital-carver/julia_brave_goggle

Submitting the URL https://github.com/digital-carver/julia_brave_goggle/raw/main/julia_programming.goggle at https://search.brave.com/goggles/create?nav=site shows just "There was an error processing your Goggle", with no further details. 

![Screenshot_20230428_164635](https://user-images.githubusercontent.com/28880/235134267-083dd6e3-0fac-49eb-a5f2-8bc09d99b46a.png)

**Comment by remusao** [comment]
Hi @digital-carver,

Thanks for reaching out. This might have been a temporary issue on the system since I was able to register your Goggle successfully. We will look into it.

https://search.brave.com/goggles/discover?goggles_id=https%3A%2F%2Fraw.githubusercontent.com%2Fdigital-carver%2Fjulia_brave_goggle%2Fmain%2Fjulia_programming.goggle&nav=site

![image](https://user-images.githubusercontent.com/1299873/235140399-45046fe8-2d08-488c-88a1-ba0f8711d880.png)

**Comment by digital-carver** [comment]
(To be clear, I'm able to use the Goggle now. I was just testing it out to see if it was a temporary issue, and I also have some changes I'd like to make and be able to resubmit the Goggle.)

**Comment by remusao** [comment]
Do you have WebAssembly and JavaScript enabled in your browser?

**Comment by digital-carver** [comment]
Ok, I figure out the issue - I compared the URLs we submitted, from the screenshots above, and realized that right-click -> Copy Link on the "Raw" button in Github doesn't give the same URL as the one it actually ends up at. 

Right cilck -> copy gives `https://github.com/digital-carver/julia_brave_goggle/raw/main/julia_programming.goggle` (which doesn't work when pasted) whereas clicking through leads to `https://raw.githubusercontent.com/digital-carver/julia_brave_goggle/main/julia_programming.goggle`. 

Pasting the latter into the Create new Goggle page works fine, so I'll be able to update the goggle now. 

The URL I was using seems to be a 302 redirect to the `raw.githubusercontent.com` page, and it seems like that isn't supported by the Goggle submit page. If a more informative error message for such cases is possible, that would be great to avoid some confusion.
